main/src/main/scala/org/clulab/numeric/mentions/package.scala

```diff
@@ -327,11 +327,11 @@ package object mentions {
         case _ => (None, None)
       }
 
-      DateRangeMention(
-        m,
-        TempEvalFormatter.mkDate(seasonNorm.get.startDay, seasonNorm.get.startMonth,yearStart),
-        TempEvalFormatter.mkDate(seasonNorm.get.endDay, seasonNorm.get.endMonth, yearEnd)
-      )
+      val sd = TempEvalFormatter.mkDate(seasonNorm.get.startDay, seasonNorm.get.startMonth, yearStart)
+      val ed = TempEvalFormatter.mkDate(seasonNorm.get.endDay, seasonNorm.get.endMonth, yearEnd)
+      val dm = DateRangeMention(m, sd, ed)
+
+      dm
```

Comment:
Somebody is going to sneak in here and give these variables real names ;-)

Reply:
:)